openedx/core/djangoapps/oauth_dispatch/dot_overrides.py

```diff
@@ -11,8 +11,10 @@
 from oauth2_provider.models import AccessToken
 from oauth2_provider.oauth2_validators import OAuth2Validator
 from pytz import utc
+import logging
 
 from .models import RestrictedApplication
+log = logging.getLogger(__name__)
```

Comment:
Add a newline above this to improve legibility. Write your code as if you were writing a manuscript. There are paragraphs to separate ideas. Imports are separate from variable declarations.